content/src/content/jcr_root/apps/core/wcm/components/form/button/v2/button/amp.html

```diff
@@ -0,0 +1,19 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<input data-sly-use.button="com.adobe.cq.wcm.core.components.models.form.Button"
+       type="submit"
```

Comment:
@gaetanmarmasse - do you know the reason why we force the `submit` type for AMP form buttons? This could lead to unexpected behaviour if the original form button isn't configured as a submit button.

Reply:
Good catch! Should be addressed by https://github.com/adobe/aem-core-wcm-components/pull/861/commits/f89a7e605f611558dcbce56fd4f3a46856f9d40e.
There is no need for an AMP specific button markup.